docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/content/home_landing/field--paragraph--field-mm-feature-card.html.twig

```diff
@@ -1,5 +1,7 @@
 <div id="nvcgSlLayoutFeatureB" class="equalheight small-4 columns card">
-  {% for item in items %}
-    {{- item.content -}}
-  {% endfor %}
+  <div class="slot-item only-SI">
```

Comment:
Can you move this into the for each? The idea is that `nvcgSlLayoutFeatureB` is the slot, and each content item in the slot gets the wrapper.

`slot-item` is given to all items in the container.
`first-SI` is given to the first element when there is more than one
`last-SI` is given to the first element when there is more than one
`only-SI` is given when it is the only element.

Maybe if someone looks at the CSS to find first/only/last and sees nothing then we can dispatch with that class.

Reply:
Ok, I have moved the slot-item wrapper into the for-each for the feature card and the media item.  The only-SI doesn't do anything in the context of the multimedia so I remvoed it.  I found no reference to the other SIs in the multimedia styles